Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+minutes'''.
+This leaves you with some room for error, and handle services that weren't well understood by their owners.
+|}
+
+=== Getting started (pre-work) ===
+
+* Ensure no previous RRA exist, else re-use the previous RRA.
+* Create a copy of the [https://drive.google.com/open?id=160V89R-VdIe1AEHcT_sX89UToV2gv8mLshhoia2-6mM template] and move it to the correct directory (or your personal Google drive if you're testing the RRA process).
+* Invite 1 or 2 members related to the service with enough technical knowledge, ensure they will bring a diagram of data
+flows and have an understanding of the data being stored or processed by the service. You do not want more than 4 or 5 persons total as this will slow down the RRA. Most RRAs are run 1 on 1 (2 persons total).
+Make sure everyone invited has '''edit''' rights to the document, and has the document opened in front of them when the
+RRA starts.
+* If this is anyone's first RRA, ensure they understand what risk impacts are, and the standard risk levels we are using, as well giving them a short overview of what is going to happen:
+** Filling in header/meta-data information about the service.
+** Getting an idea of how the service is architected.
+** Recording all data the service may process/store and how sensitive it is.
+** Run through the risk table and figure out what happens if the data leak, is modified, is unavailable, etc - while
+assigning the standard risk levels.
+** Write down any recommendation for the service.
+
+{| class="wikitable"
+|-
+! <span style="color:red;">First time?</span>
+|-
+|
+If this is your first RRA, ensure that someone who has run RRAs previously is present to help you.
+It is good to have attended multiple RRAs before starting your own.
+|}
+
+=== Header (5-10 minutes) ===
+
+The header has to be filled in first and foremost.
+
+* Fill in the service name. It may be the product or generic service name, or even better, both. Try to keep it short and simple.
+* Fill in the description of the service. In order to do that, ask the team to give you a short, one sentence, few words quick description of what this service does, how it does it.
+* Fill in the scope if you know it. Sometimes the scope is clearer at the end of the RRA.
+** If the scope is obvious (its the whole service) it is OK to leave it blank.
+** In some cases the scope is critical to the understanding of the RRA.
+* Ask the team who owns the service (takes decisions if the service must go down for example), who develops the code and who's operating it.
+** Generally its a team name and one or more names. Sometimes, it's a third party (SaaS).
+** You never want to leave the service owner blank, and you always want a Mozilla contact as service owner.
+** You always want at least a team name, as people change teams, etc.
+* Other fields will be automatically filled in or filled in at a later time.
+
+=== General notes (5-10 minutes) ===
```

Comment:
This is where we ask people about the service and take notes, but one should keep taking notes even after this section is done, instead of blocking on it.